src/js/page.js

```diff
@@ -145,9 +147,9 @@
       let nodeSelector = "#newtab-grid, #newtab-search-container";
       for (let node of document.querySelectorAll(nodeSelector)) {
```

Comment:
Sent from my iPhone

> On Nov 6, 2015, at 3:28 AM, Ursula Sarracini <notifications@github.com> wrote:
> 
> In src/js/page.js:
> 
> > @@ -145,9 +147,9 @@
> >        let nodeSelector = "#newtab-grid, #newtab-search-container";
> >        for (let node of document.querySelectorAll(nodeSelector)) {
> @marcoscaceres can't use forEach on querySelectorAll...not an array! :(
> could do something like
> 
> var nodeList = document.querySelectorAll("#newtab-grid, #newtab-search-container"); 
> var nodeArray = Array.prototype.slice.call(nodeList);
> nodeArray.forEach(node => node.dataset.pageDisabled = !aValue);
> thoughts?
> 
Ah yeah, you are right. 
With Es6, we have 2 better options:
```js
Array.from(nodeList).forEarch();
// or
(...nodeList).forEach()
```

So we never need to use the ugly Array.prototype.slice.call(💩) hack thing anymore 🤗.
> 
> —
> Reply to this email directly or view it on GitHub.
>